src/core/secondaryAdapters/officialOnyxiaApiClient.ts

```diff
@@ -325,7 +341,7 @@ export function createOfficialOnyxiaApiClient(params: {
                     dependencies?: {
                         name: string;
                     }[];
-                }>(`/public/catalog/${catalogId}/${packageName}`)
+                }>(`${restApiPaths.catalogs}/${catalogId}/charts/${packageName}`)
```

Comment:
No need to extract this token.  
This is the source of truth and it only appear here.